contrib/examples/walkthrough/ups-instance-default-sp.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: servicecatalog.k8s.io/v1alpha1
```

Comment:
I don't feel strongly. I think in the future we will want to migrate away from using user provided services as our walkthrough example anyway, as it is a bit of a special case (potentially bundled with catalog, no auth, plan concept not making sense, doesn't tell the main "consuming third party services is easy!" OSB story).